package.json

```diff
@@ -16,7 +16,7 @@
     "react-transition-group": "2.6.0",
     "sanitize-html": "^1.18.4",
     "sass": "1.32.5",
-    "synapse-react-client": "2.0.58",
+    "synapse-react-client": "2.0.61",
```

Comment:
I'd request that we use 2.0.62 when it's out (Download List v2 is in a better place)